build/system.gyp

```diff
@@ -11,6 +11,19 @@
   },
   'targets' : [
     {
+      'target_name': 'linux_notification',
+      'type': 'none',
+      'conditions': [
+        ['tizen!=1', {
+          'link_settings': {
+            'libraries': [
+              '<!@(pkg-config --libs libnotify)',
```

Comment:
Sorry, yes, should take 'building_crosswalk_bin' into consideration, thank you very much.